Fedora specific configuration - we want to mount all controllers by default for libvirt.
 
diff --git a/samples/config/cgconfig.conf b/samples/config/cgconfig.conf
index de29594..2020d71 100644
--- a/samples/config/cgconfig.conf
+++ b/samples/config/cgconfig.conf
@@ -4,39 +4,8 @@
 #
 #  Authors:	Balbir Singh <balbir@linux.vnet.ibm.com>
 #
-#group daemons/www {
-#	perm {
-#		task {
-#			uid = root;
-#			gid = webmaster;
-#		}
-#		admin {
-#			uid = root;
-#			gid = root;
-#		}
-#	}
-#	cpu {
-#		cpu.shares = 1000;
-#	}
-#}
 #
-#group daemons/ftp {
-#	perm {
-#		task {
-#			uid = root;
-#			gid = ftpmaster;
-#		}
-#		admin {
-#			uid = root;
-#			gid = root;
-#		}
-#	}
-#	cpu {
-#		cpu.shares = 500;
-#	}
-#}
-#
-#mount {
-#	cpu = /mnt/cgroups/cpu;
-#	cpuacct = /mnt/cgroups/cpuacct;
-#}
+# By default, we expect systemd mounts everything on boot,
+# so there is not much to do.
+# See man cgconfig.conf for further details, how to create groups
+# on system boot using this file.
diff --git a/samples/config/cgconfig.sysconfig b/samples/config/cgconfig.sysconfig
index 5a61bf6..0f4f6d1 100644
--- a/samples/config/cgconfig.sysconfig
+++ b/samples/config/cgconfig.sysconfig
@@ -5,8 +5,6 @@
 # controller to limit cpu.shares of this default group and allowing some more
 # important group take most of the CPU.
 #
-# By default, create these groups:
-CREATE_DEFAULT=yes
-
-# Uncomment following line to disable creation of the default group on startup:
-# CREATE_DEFAULT=no
+# By default, do not create these groups:
+# CREATE_DEFAULT=yes
+CREATE_DEFAULT=no
